src/test/java/seedu/address/commons/core/ConfigTest.java

```diff
@@ -20,7 +24,31 @@ public void toString_defaultObject_stringReturned() {
     public void equalsMethod() {
         Config defaultConfig = new Config();
         assertNotNull(defaultConfig);
+
+        // same values -> returns true
+        Config copy = new Config();
+        copy.setLogLevel(defaultConfig.getLogLevel());
+        copy.setUserPrefsFilePath(defaultConfig.getUserPrefsFilePath());
+        assertTrue(defaultConfig.equals(copy));
+
+        // same object -> returns true
         assertTrue(defaultConfig.equals(defaultConfig));
+
+        // null -> returns false
+        assertFalse(defaultConfig.equals(null));
+
+        // different types -> returns false
+        assertFalse(defaultConfig.equals(5));
+
+        // different level -> returns false
```

Comment:
A really minor comment: would it be better to make the comment singular/plural form consistent?

Reply:
Yep, I agree. Already updated it on my end.